deps.ffmpeg/60-srt.zsh

```diff
@@ -140,6 +152,11 @@ fixup() {
       }
 
       autoload -Uz restore_dlls && restore_dlls
+      
+      # TODO: To be removed once upgraded to 1.6.0
+      # ref: https://github.com/Haivision/srt/pull/2536
+      log_info "Fixup pkgconfig"
+      sed -i 's/-l-l:libunwind.a //g' ${target_config[output_dir]}/lib/pkgconfig/srt.pc
```

Comment:
Use a patch file with proper link to the GitHub issue and corresponding file name instead.